scripts/linters/general_purpose_linter.py

```diff
@@ -188,7 +188,8 @@
             '\n'
             'class SampleModel(datastore_services.Model):\n'
             '    ...\n'),
-        'excluded_files': (),
+        'excluded_files': (
+            'scripts/check_backend_associated_test_file.py',),
```

Comment:
Why is this added?

Reply:
I added this because the file `scripts/linters/test_files/invalid_ndb.py` was listed in `FILES_WITHOUT_ASSOCIATED_TEST_FILES` and was throwing the error:

![Screenshot from 2022-06-29 10-21-13](https://user-images.githubusercontent.com/89375125/176354462-4fbb85dd-720f-42ea-996d-cb44da8415d0.png)

I referred to Chris for the same and he suggested we could add the file in lint exclusion list.

Reply:
Ah, can we rather rename the invalid_ndb file to smth invalud_datastore?

Reply:
Done!

Reply:
Not sure.